src/coreclr/jit/lsraxarch.cpp

```diff
@@ -154,12 +154,14 @@ int LinearScan::BuildNode(GenTree* tree)
         case GT_CNS_VEC:
         {
             srcCount = 0;
+
             assert(dstCount == 1);
             assert(!tree->IsReuseRegVal());
-            RefPosition* def               = BuildDef(tree, BuildEvexIncompatibleMask(tree));
```

Comment:
Why is `BuildEvexIncompatibleMask()` not needed anymore?

Reply:
It was added early before the EVEX support was fully online to avoid breaking `TYP_SIMD16`/`TYP_SIMD32` code paths.

Constants can be (and are required to be for `TYP_SIMD64`) in EVEX registers. The user being one of the handful of evex incompatible instructions is the only thing that could restrict it otherwise.

We've had full EVEX support on for a while now, so removing this allows LSRA to better take advantage of the hardware.

Reply:
-- There was one case that did need a fixup around `AllBitsSet` as it needs to generate `vpternlogd` rather than `pcmpeqd` in that scenario. I've added a commit which handles that